rust/headless-client/src/imp_linux.rs

```diff
@@ -29,11 +31,19 @@ const ROOT_USER: u32 = 0;
 /// on some systems, `/run` should be the newer version.
 const SOCK_PATH: &str = "/run/firezone-client.sock";
 
+pub fn default_token_path() -> PathBuf {
+    PathBuf::from("/etc")
+        .join(connlib_shared::BUNDLE_ID)
+        .join("token.txt")
```

Comment:
(nit): I don't think it's common for plaintext files on Linux to have `.txt` is it? Especially in the `/etc` directory?

Reply:
You're right, going by the precedent of SSH privkeys, I'll remove the extension